benchmark/attention_training/configs/llama.py

```
@@ -31,7 +31,7 @@
         (4096, 4096),
         (2048, 2048),
     ],
-    backends=["cudnn", "cudnn_oss", "flash_attention_4"],
+    backends=["cudnn", "cudnn_oss", "flash_attention_4"] + fa2_on_ampere(),
```

Ruff (0.16.3)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)